src/redux/actions/projectActions.ts

```diff
@@ -84,11 +84,24 @@ export function saveProject(project: IProject):
  * Dispatches Delete Project action and resolves with project
  * @param project - Project to delete
  */
-export function deleteProject(project: IProject): (dispatch: Dispatch) => Promise<void> {
-    return async (dispatch: Dispatch) => {
+export function deleteProject(project: IProject)
+    : (dispatch: Dispatch, getState: () => IApplicationState) => Promise<void> {
+    return async (dispatch: Dispatch, getState: () => IApplicationState) => {
+        const appState = getState();
         const projectService = new ProjectService();
-        await projectService.delete(project);
-        dispatch(deleteProjectAction(project));
+
+        // Lookup security token used to decrypt project settings
+        const securityToken = appState.appSettings.securityTokens
+            .find((st) => st.name === project.securityToken);
```

Comment:
I prefer `securityToken` to `st`

Reply:
I agree, but didn't want to have 2 scoped variables with the same name since `securityToken` is defined above.  I will consider renaming both of them.